includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -49,4 +52,46 @@ public static function track_store_setup() {
 
 		WC_Tracks::record_event( 'obw_store_setup', $properties );
 	}
+
+	/**
+	 * Track payment gateways selected.
+	 *
+	 * @return void
+	 */
+	public static function track_payments() {
+		$selected_gateways     = array();
+		$created_accounts      = array();
+		$wc_admin_setup_wizard = new WC_Admin_Setup_Wizard();
+		$gateways              = array_merge( $wc_admin_setup_wizard->get_wizard_in_cart_payment_gateways(), $wc_admin_setup_wizard->get_wizard_manual_payment_gateways() );
+
+		foreach ( $gateways as $gateway_id => $gateway ) {
+			if ( ! empty( $_POST[ 'wc-wizard-service-' . $gateway_id . '-enabled' ] ) ) { // WPCS: CSRF ok, input var ok.
+				$selected_gateways[] = $gateway_id;
+			}
+		}
+
+		// Stripe account being created.
+		if (
+			! empty( $_POST['wc-wizard-service-stripe-enabled'] ) && // WPCS: CSRF ok, input var ok.
+			! empty( $_POST['stripe_create_account'] ) // WPCS: CSRF ok, input var ok.
+		) {
+			$created_accounts[] = 'stripe';
+		}
+		// PayPal account being created.
+		if (
+			! empty( $_POST['wc-wizard-service-ppec_paypal-enabled'] ) && // WPCS: CSRF ok, input var ok.
+			! empty( $_POST['ppec_paypal_reroute_requests'] ) // WPCS: CSRF ok, input var ok.
+		) {
+			$created_accounts[] = 'ppec_paypal';
+		}
+
+		// phpcs:disable WordPress.Security.NonceVerification.NoNonceVerification, WordPress.Security.ValidatedSanitizedInput
+		$properties = array(
+			'selected_gateways' => $selected_gateways,
```

Comment:
These arrays get tracked with individual properties and indexes.  I assume this is the best way to grok this info later on, but can someone confirm this is accurate or if we need to switch to a comma separated string?

<img width="364" alt="screen shot 2019-02-27 at 1 58 04 pm" src="https://user-images.githubusercontent.com/10561050/53469367-9160f000-3a98-11e9-9e20-be7be16c5a62.png">

Reply:
Looks like comma separated string is the way to go. A funnel step can be created via event properties which allow a regex.

Reply:
Awesome- thanks for checking on that @psealock!  Updated in https://github.com/woocommerce/woocommerce/pull/22872/commits/7ad7de55a1d6d93a9ca76e3e289b8fd8e282d93f